lib/pbench/common/selinux.py

```diff
@@ -0,0 +1,36 @@
+"""SELinux wrapper module.
+
+There is no "pip" installable `selinux` python module, just a shim, of sorts [1],
+which provides a way to add the installed system's `selinux` module to a
+virtualenv.  But that does not work when the virtualenv uses a version of
+python different from the python version where the real `selinux` module is
+installed.
```

Comment:
If you edit this module again, it would be nice to clarify that the "installed system's" module got there (e.g.) via RPM (because, typically, Pip is how Python modules get "installed"...and without the RPM prompt, these statements are likely to be confusing).